### PR TITLE
Create folderSeperator.py

### LINKED ISSUE #221: Folder Separator
Separates files into the respective folder

**Is your feature request related to a problem? Please describe.**
It is difficult to separate files into different folders

**Describe the solution you'd like**
Separates files into the respective folder

**Additional context**
No

**Languages**

-   [x] Python
-   [ ] Java
-   [ ] C
-   [ ] C++

### PR DESCRIPTION
**Pull request by MayankDev-11**
# Title/Heading < File seperator>

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. < This is a File seperator>

Fixes #221

## Type of change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

# Checklist:

-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generates no new warnings
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] New and existing unit tests pass locally with my changes
-   [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: Python/folderSeperator.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import os
```
Comment (kaustubhgupta) [inline Python/folderSeperator.py:1]: Please update the readme of the language folder

File: Python/folderSeperator.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+others = []
+
+for file in files:
+    ext = os.path.splitext(file)[1].lower()
+    if (ext not in mediaExts) and (ext not in imgExts) and (ext not in docExts) and (ext not in pyExts) and os.path.isfile(file):
+        others.append(file)
+
+
+print("These are other folders:","\n",others,"\n")
+
```
Comment (kaustubhgupta) [inline Python/folderSeperator.py]: Remove this extra line

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Python/folderSeperator.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+def move(foldername,files):
+    for file in files:
+        os.replace(file, f"{foldername}/{files}")
+        print("Ok")
+
+
+files = os.listdir()
+files.remove("folder_Cleaner.py")
```
Comment (ByGreez) [inline Python/folderSeperator.py:16]:
```suggestion
files.remove("folderSeperator.py")
```

**Comment by MayankDev-11** [comment]
Yes bro i will add this feature too.Working on it

**Comment by kaustubhgupta** [comment]
@MayankDev-11 If you fail to make changes within 7 days of opening the PR it will be marked as stale and on the 10th day it will be closed automatically, just an update

File: Python/folderSeperator.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import os
```
Comment (kaustubhgupta) [inline Python/folderSeperator.py:1]: Update readme folder of the Python folder with your entry

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by kaustubhgupta** [comment]
Due to inactivity, this PR has been closed

**Pull request closed without merging**